package.json

```diff
@@ -19,6 +19,7 @@
     "prepare": "husky install",
     "lint": "eslint . --fix",
     "test": "jest --maxWorkers=100%",
+    "sanity": "jest --maxWorkers=100% --config=jest.sanity.js",
```

Comment:
Do you want to add --testTimeout=<number>?
I run your test and I get this
<img width="793" alt="Screenshot 2022-05-18 at 1 02 13 AM" src="https://user-images.githubusercontent.com/7101806/168870191-c3b5a42f-8018-4966-a61f-99ba480b2360.png">

Reply:
Build times can definitely be excessive in some cases, what do we think is a sensible timeout value?